docs/custom.css

```
@@ -231,6 +231,248 @@
+
+  /* Themed aliases (light default, dark override below) */
+  --mastra-color-surface-bg: var(--mastra-color-light-surface-bg);
+  --mastra-color-surface-antigrid: var(--mastra-color-light-surface-antigrid);
+  --mastra-color-surface-elevation-sm: var(--mastra-color-light-surface-elevation-sm);
+  --mastra-color-surface-elevation-lg: var(--mastra-color-light-surface-elevation-lg);
+  --mastra-color-surface-hover: var(--mastra-color-light-surface-hover);
+
+  --mastra-color-border-antigrid: var(--mastra-color-light-border-antigrid);
+  --mastra-color-border-elevation: var(--mastra-color-light-border-elevation);
+
+  --mastra-shadow-dropdown: 0 4px 54px 0 rgba(0, 0, 0, 0.12);
```

Stylelint (17.14.0)

[error] Expected "0.12" to be "12%" (alpha-value-notation)

(alpha-value-notation)